manage.py

```diff
@@ -2,10 +2,14 @@
 import os
 import sys
 
-from newrelic import agent
+try:
+    from newrelic import agent
+except ImportError:
+    agent = None
```

Comment:
Given that you've removed `newrelic`, when do you expect the import to ever succeed?